flutter_vlc_player/pigeons/messages.dart

```diff
@@ -2,6 +2,7 @@ import 'package:pigeon/pigeon_lib.dart';
 
 class ViewMessage {
   int? viewId;
+  bool isTexture = false;
```

Comment:
is there a way to avoid this isTexture repeats in those messages? Maybe inheritance or pass it in init somehow?

Reply:
It seems Pigeon does not generate the fields of the parent class in case of inheritance.

Reply:
> It seems Pigeon does not generate the fields of the parent class in case of inheritance.

Try the latest version https://pub.dev/packages/pigeon